src/main/java/org/mockito/internal/matchers/Equals.java

```diff
@@ -5,16 +5,16 @@
 
```

Comment:
Don't get me wrong, these changes are good, but should be targeted by an other pull request.